conf/prod.routes

```diff
@@ -3,6 +3,9 @@
 ->         /setup                   setup.Routes
 ->         /email                   email.Routes
 ->         /secure-message          secureMessage.Routes
+->         /secure-comms-alert      secureCommsAlert.Routes
+->         /hmrc/email              email.Routes
+->         /messages                secureMessage.Routes
```

Comment:
If `/messages` is the new route prefix for `secureMessage.Routes` then I'd remove the other one to avoid confusion.